### DISCUSSION TITLE
Update code examples to next.js 15

### DISCUSSION THREAD

**Discussion by pieterjanscheir** [amannn/next-intl#1612] [General]
Hello, 

Could the code examples be updated to the newest version of Next.js. This would increase the confidence level of implementators of your package to use next-intl.

Is there a reason why this hasn't happened yet?

Thank you in advance and thanks for the very nice package you've provided!

Kind regards,

Pieter-Jan

**Comment by amannn** [comment]
I'm currently waiting for an upstream bug to be fixed before I'd like to recommend Next.js 15, please see https://github.com/amannn/next-intl/pull/1089 for details! 🙂

**Reply by pieterjanscheir** [reply]
Oh, sorry, didn't found the duplicate before. Thank you for the info!
